navlie/lib/models.py

```diff
@@ -714,6 +719,78 @@ def jacobian(self, x: MatrixLieGroupState) -> np.ndarray:
     def covariance(self, x: MatrixLieGroupState) -> np.ndarray:
         return self._R
 
+class PointRelativePositionSLAM(MeasurementModel):
```

Comment:
Can we add unit test for this class? :) Apart from that, LGTM!

Reply:
Unit test added!!